tests/distributions/test_transformed_distribution.py

```diff
@@ -51,7 +52,9 @@ def AffineExp(affine_b, affine_a):
 
 
 def get_transformed_dist(distribution, affine_a, affine_b):
-    return TransformedDistribution(distribution, [AffineExp(affine_b, affine_a)])
+    # Set `validate_args=False` as base distribution would have been validated, if needed.
```

Comment:
I'd really like to avoid this workaround after #928 fixes the bug in `TransformedDistribution`.

Reply:
Makes sense. Let us merge the fixes from #928 first, and then update this PR.